src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -465,22 +497,65 @@ class CosmasGoogleCloudService(private val bucketName: String,
         responseObserver.onCompleted()
     }
 
+    override fun changeUserPlan(request: ChangeUserPlanRequest,
+                                responseObserver: StreamObserver<ChangeUserPlanResponse>) {
+        changeUserPlan(request, responseObserver, Runtime.getRuntime())
+    }
+
+    fun changeUserPlan(request: ChangeUserPlanRequest,
+                       responseObserver: StreamObserver<ChangeUserPlanResponse>,
+                       runtime: Runtime) {
+        val isFreePlanNow = request.isFreePlanNow // plan AFTER changing
+        if (isFreePlanNow) {
+            LOG.info("Get request for change user # ${request.userId} plan from paid to free")
+        } else {
+            LOG.info("Get request for change user # ${request.userId} plan from free to paid")
+        }
+        val oldBucketName = bucketName(!isFreePlanNow)
+        val newBucketName = bucketName(isFreePlanNow)
+        val gsutilCopyCommand = "gsutil cp -r -A gs://$oldBucketName/${request.userId}/*" +
```

Comment:
It is unlikely that we'll have gsutil in the production Docker image with Cosmas. We will have another Docker image with gsutil though, so in production we'll need to run something like `docker run google/cloud-sdk gsutil cp -r ....`. 

We have a number of options with respect to this:
1. Move command line with placeholders to config  and run gsutil or docker depending on the value in config -- this requires writing a few lines of code and requires `docker` client to be in the Docker image with Cosmas
1. Add our own `gsutil` script to Cosmas docker image which will just run `docker run...` with all passed arguments. No changes in this code, but also docker client in Cosmas image + our own script
1. Run the equivalent of `docker run` using Java API for docker. This will require no changes in Cosmas docker image but will require writing, basically, another implementation of this `changeUserPlan` function.

Reply:
done (1 option)